.travis.yml

```diff
@@ -87,8 +88,13 @@ before_script:
 
 # Run the tests.
 script:
-  - "make tests PYENV= SHELL=$BUILDS_PATH/$ZSH_BUILD_VERSION/zsh"
-  - "make stats _ANTIGEN_CACHE_ENABLED=false _ANTIGEN_INIT_ENABLED=false SHELL=$BUILDS_PATH/$ZSH_BUILD_VERSION/zsh"
-  - "make stats _ANTIGEN_CACHE_ENABLED=true _ANTIGEN_INIT_ENABLED=false SHELL=$BUILDS_PATH/$ZSH_BUILD_VERSION/zsh"
-  - "make stats _ANTIGEN_CACHE_ENABLED=true _ANTIGEN_INIT_ENABLED=true SHELL=$BUILDS_PATH/$ZSH_BUILD_VERSION/zsh"
+  - "make tests SH=$ZSH_BIN"
+  - "make stats _ANTIGEN_CACHE_ENABLED=false SH=$ZSH_BIN"
+  - "make stats _ANTIGEN_CACHE_ENABLED=true SH=$ZSH_BIN"
+  - "make stats _ANTIGEN_CACHE_ENABLED=true STATS_USE_INIT=true SH=$ZSH_BIN"
+  # Test reproducible build across osx and gnu
```

Comment:
Could I suggest:

> Ensure the checked-in build matches the build produced in CI. Since the CI will run this on both OS X and Linux, the two platforms must also produce the _same_ build, else one of them will fail.